### PR TITLE
Standardize Copyright Disclaimer in *.cs Files

### PR DESCRIPTION
**Pull request by MeenuyD**
In alignment with OpenTelemetry practices (examples https://github.com/open-telemetry/opentelemetry-demo/blob/f44860602656e2313f3cfe6f7e42a464a341786b/src/cartservice/src/Program.cs#L1C1-L2C39 and 
 https://github.com/open-telemetry/opentelemetry-cpp/blob/ad626cec4598165e0d297b0bdb3880968c5dfe88/api/include/opentelemetry/config.h#L1-L2), simplified the copyright disclaimer in all *.cs files to the format:

// Copyright The OpenTelemetry Authors
// SPDX-License-Identifier: Apache-2.0
This standardizes the copyright information and aligns it with our common practices across OpenTelemetry projects.
#4900

### CODE REVIEW

**Comment by MeenuyD** [comment]
Made changes to standardize the copyright disclaimer in the selected *.cs files. If it meets the project's standards, I am prepared to apply the same changes across all *.cs files upon your approval. The updated format is:

```

// Copyright The OpenTelemetry Authors
// SPDX-License-Identifier: Apache-2.0

```


Please review, and I'll proceed accordingly."

**Comment by CodeBlanch** [comment]
@MeenuyD This is great! Please continue.

**Comment by reyang** [comment]
@MeenuyD FYI, https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/build/stylecop.json needs to be changed to:

```xml
{
  "$schema": "https://raw.githubusercontent.com/DotNetAnalyzers/StyleCopAnalyzers/master/StyleCop.Analyzers/StyleCop.Analyzers/Settings/stylecop.schema.json",
  "settings": {
    "documentationRules": {
      "companyName": "OpenTelemetry Authors",
      "copyrightText": "Copyright The OpenTelemetry Authors\nSPDX-License-Identifier: Apache-2.0",
      "xmlHeader": false
    },
    "orderingRules": {
      "usingDirectivesPlacement": "outsideNamespace"
    }
  }
}
```

File: build/stylecop.json [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
-      "copyrightText": "Copyright The OpenTelemetry Authors\n\nLicensed under the Apache License, Version 2.0 (the \"License\");\nyou may not use this file except in compliance with the License.\nYou may obtain a copy of the License at\n\n    http://www.apache.org/licenses/LICENSE-2.0\n\nUnless required by applicable law or agreed to in writing, software\ndistributed under the License is distributed on an \"AS IS\" BASIS,\nWITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\nSee the License for the specific language governing permissions and\nlimitations under the License."
+      "copyrightText": "Copyright The OpenTelemetry Authors\nSPDX-License-Identifier: Apache-2.0",
+      "xmlHeader": false
     },
     "orderingRules": {
       "usingDirectivesPlacement": "outsideNamespace"
     }
   }
-}
+}
```
Comment (reyang) [inline build/stylecop.json:12]: nit: add an empty line before EOF

**Comment by utpilla** [comment]
@MeenuyD Could you please update the branch and fix the failing tests?

**Review by reyang — APPROVED** [review]
LGTM assuming all *.cs files will be updated.

**Comment by MeenuyD** [comment]
Should I do it now

**Comment by reyang** [comment]
> Should I do it now

Yes please. Thanks!!!

**Comment by MeenuyD** [comment]
Can you please review the #4930

**Pull request closed without merging**